src/mono/wasm/debugger/DebuggerTestSuite/SteppingTests.cs

```diff
@@ -948,5 +948,25 @@ public async Task CheckResetFrameNumberForEachStep()
             pause_location = await StepAndCheck(StepKind.Over, "dotnet://debugger-test.dll/debugger-test.cs", 806, 8, "Increment");
             Assert.Equal(pause_location["callFrames"][0]["callFrameId"], "dotnet:scope:1");
         }
+
+        [Fact]
+        public async Task DebuggerAttributeIgnoreStepIntoDebuggerHidden()
+        {
+            var pause_location = await SetBreakpointInMethod("debugger-test.dll", "DebuggerAttribute", "Run", 1);
+            await EvaluateAndCheck(
+                "window.setTimeout(function() { invoke_static_method('[debugger-test] DebuggerAttribute:Run'); }, 1);",
+                "dotnet://debugger-test.dll/debugger-test.cs",
+                pause_location.Value["locations"][0]["lineNumber"].Value<int>(),
+                pause_location.Value["locations"][0]["columnNumber"].Value<int>(),
+                "Run"
+            );
+            var step_into = await SendCommandAndCheck(null, $"Debugger.stepInto", null, -1, -1, null);
+
+            Assert.Equal(
+                step_into["callFrames"][0]["location"]["lineNumber"].Value<int>(), 
```

Comment:
As we discussed offline, the step_into should be on the next line.

Reply:
I still need to correct the code so that the test will be passing.